java/README.md

```diff
@@ -2,57 +2,58 @@
 
 ## Verification Task Structure
 
-Verification tasks are grouped in directories, e.g.,
-`jbmc-regression`.  Within these directories, each verification task
-is in its own directory, e.g., `StringValueOf01_true-assert`. The
-suffix of this directory contains the expected verification outcome,
-i.e., `_true-assert` means that there is no execution of the program
-that violates any of the assertions in that program.  A verification
-task has a structure compatible with the [Maven](https://maven.apache.org)
-standard directory layout (to allow the future use of the Maven build
-system if necessary):
-
-| directory | description |
-| --------- | ----------- |
-|`src/` | |
-| `src/main/java/Main.java` | The `Main` class. |
-| `src/main/java/` | This directory contains the remaining Java source tree. |
-| `target/` | |
-| `target/classes/` | After compiling this directory contains the `.class` built from `src`. |
-| `target/task.jar` | This file assembles the `.class` files from `target/classes`. |
+Verification tasks are grouped in directories depending on their source,
+e.g., `jbmc-regression`.
+Within these directories, each verification task consists of a YAML file
+in the format defined by BenchExec for task-definition files.
+These YAML files define the list of input files (Java sources) of a task
+and the expected verdict for each possible property.
 
 The programs are assumed to be written in Java 1.8.
-The `.java` source files are in the source tree
-in the `src` sub-directory of the benchmark directory.  The
-program may call the Java standard library (`java.*`,
-`javax.*`).  The benchmark must have a `Main` class with a
```

Comment:
This sentence needs to be kept: "The program may call the Java standard library (`java.*`, `javax.*`)."

Reply:
There is no need to require that the source tree starts at the root of the benchmark dir. If there is a benchmark in the future that contains additional files, it can make sense to deviate. Examples:
- source code originally in a different language, and transpiled to Java
- source code with several directories (`src` and `test`)

There is also no need to have a `Main` class: benchmarks with a different entry point just need to refer to a different property file.

The sentence about the Java standard library is still there.